books/centaur/sv/svtv/process.lisp

```diff
@@ -1559,30 +1559,60 @@ stvs-and-testing) of the @(see sv-tutorial) for more examples.</p>"
            :hints(("Goal" :in-theory (enable svex-alist-p)))))
 
   (defthm lookup-in-svtv-run-under-iff
-    (iff (assoc key (svtv-run svtv inalist :skip skip :boolvars boolvars :quiet quiet))
+    (iff (assoc key (svtv-run
+                     svtv inalist
+                     :include include :skip skip :boolvars boolvars
+                     :simplify simplify :quiet quiet :readable readable))
          (and (member key (svtv->outs svtv))
-              (not (member key skip))))
+              (if include
+                  (member key include)
+                (not (member key skip)))))
     :hints(("Goal" :in-theory (enable svtv->outs))))
 
   (defthm lookup-in-svtv-run-consp
-    (equal (consp (assoc key (svtv-run svtv inalist :skip skip :boolvars boolvars :quiet quiet)))
-         (and (member key (svtv->outs svtv))
-              (not (member key skip))))
+    (equal (consp
+            (assoc key (svtv-run
+                        svtv inalist
+                        :include include :skip skip :boolvars boolvars
+                        :simplify simplify :quiet quiet :readable readable)))
+           (and (member key (svtv->outs svtv))
+                (if include
+                    (member key include)
+                  (not (member key skip)))
+                t))
```

Comment:
The only problem with (and (foo) t) is that it will casesplit on (foo).  But in this case I think we're anticipating that we'll mostly see calls of svtv-run with constants for skip and include so it should be OK.  Otherwise we could use (consp (member ...)) or just change the equiv relation to IFF.